src/tcgwars/logic/impl/gen8/SwordShieldPromos.groovy

```diff
@@ -835,6 +875,298 @@ public enum SwordShieldPromos implements LogicCardInfo {
+      case ELDEGOSS_V_SWSH84:
+      return copy (RebelClash.ELDEGOSS_V_19, this)
+      case BOLTUND_V_SWSH85:
+      return copy (RebelClash.BOLTUND_V_67, this)
+      case CRAMORANT_V_SWSH86:
+      return copy (SwordShield.CRAMORANT_V_155, this)
+      case EEVEE_VMAX_SWSH87:
+      return evolution (this, from: "Eevee V", hp:HP300, type:C, retreatCost:2) {
+        weakness F
+        move "G-Max Cuddle", {
+          text "During your opponent’s next turn, if the Defending Pokémon tries to attack, your opponent flips a coin. If tails, that attack doesn’t happen."
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            // TODO
+          }
+        }
+      }
+      case CHERRIM_SWSH88:
+      return copy (BattleStyles.CHERRIM_8, this)
+      case OCTILLERY_SWSH89:
+      return copy (BattleStyles.OCTILLERY_37, this)
+      case HOUNDOOM_SWSH90:
+      return copy (BattleStyles.HOUNDOOM_96, this)
+      case BRONZONG_SWSH91:
+      return copy (BattleStyles.BRONZONG_102, this)
+      case CHARMANDER_SWSH92:
+      return copy (VividVoltage.CHARMANDER_23, this)
+      case ARROKUDA_SWSH93:
+      return copy (VividVoltage.ARROKUDA_41, this)
+      case JOLTEON_SWSH94:
+      return evolution (this, from: "Eevee", hp:HP100, type:L, retreatCost:0) {
+        weakness F
+        move "Energize", {
+            text "Attach a [L] Energy card from your discard pile to this Pokémon."
+            energyCost L
+            attackRequirement {
+              assert my.discard.filterByEnergyType(L) : "No [L] Energy in your discard pile."
+            }
+            onAttack {
+              afterDamage {
+                attachEnergyFrom(type: L, my.discard, self)
+              }
+            }
+          }
+        move "Thunder", {
+          text "160 damage. This Pokémon also does 30 damage to itself."
+          energyCost L, L, C
+          attackRequirement {}
+          onAttack {
+            damage 160
+            damage 30, self
+          }
+        }
+      }
+      case EEVEE_SWSH95:
+      return basic (this, hp:HP060, type:C, retreatCost:1) {
+        weakness F
+        move "First Step", {
+          text "Draw a card."
+          energyCost C
+          attackRequirement {
+            assert my.deck "There are no cards in your deck."
+          }
+          onAttack {
+            draw 1
+          }
+        }
+        move "Tail Whap", {
+          text "30 damage."
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      }
+      case DRAGAPULT_V_SWSH96:
+      return copy (VividVoltage.ARROKUDA_41, this)
+      case DRAGAPULT_VMAX_SWSH97:
+      return copy (VividVoltage.ARROKUDA_41, this)
+      case CROBAT_V_SWSH98:
+      return copy (VividVoltage.ARROKUDA_41, this)
+      case CROBAT_VMAX_SWSH99:
+      return copy (VividVoltage.ARROKUDA_41, this)
```

Comment:
Copying the wrong card.

Reply:
Woops! I copy pasted these to come back and take care of them later, but it seems I forgot to fill these in.